accounts/ethstore/src/account/safe_account.rs

```diff
@@ -205,21 +205,21 @@ mod tests {
 
 	#[test]
 	fn sign_and_verify_public() {
-		let keypair = Random.generate().unwrap();
+		let keypair = Random.generate();
 		let password = "hello world".into();
-		let message = Message::default();
+		let message = [1u8; 32].into();
```

Comment:
In upstream `rust-secp256k1`, a [`Message`](https://github.com/rust-bitcoin/rust-secp256k1/blob/master/src/lib.rs#L449-L470) cannot be all zeroes anymore. This is one big reason why this PR is so noisy: we have a lot of code that use the `Default` trait to initialize values.